src/main/java/de/rabea/Game/GameSetUp.java

```diff
@@ -0,0 +1,43 @@
+package de.rabea.game;
+
+import de.rabea.ui.RealConsole;
+import de.rabea.ui.UserInterface;
+
+public class GameSetUp {
+    UserInterface userInterface = new UserInterface(new RealConsole(System.in, System.out));
+
+    public GameSetUp(UserInterface userInterface) {
+        this.userInterface = userInterface;
+    }
+
+    public void setUpFirstGame() {
+        userInterface.greet();
+        setUpGame();
+    }
+
+    public void setUpGame() {
+        GameMode gameMode = userInterface.chooseGameMode();
+        userInterface.announceMarkDistribution(gameMode);
+
+        if (gameMode == GameMode.HvC) {
+            Game game = new Game(userInterface, createNewHumanPlayer(), createNewComputerPlayer(), this);
+            game.play();
```

Comment:
game.play is duplicated here, and the two bodies of the if loop are v.similar. You could pull it outside the if loops, or try to isolate the parts that change so you don't need an if loop at all.